ceph/monitors_test.go

```diff
@@ -266,15 +268,18 @@ func TestMonitorCollector(t *testing.T) {
 		},
 	} {
 		func() {
-			conn := &MockConn{}
+			conn := setupVersionMocks(tt.version, "{}")
 			conn.On("MonCommand", mock.Anything).Return(
```

Comment:
I'm not super familiar with the mocking going on here, but does this override what you're setting up in `setupVersionMocks`? Or do they work in sequence, i.e. the first call to MonCommand is going to return the line below this one, and the next two will return what's in `setupVersionMocks`?

Reply:
Oh I see, the two in setupVersionMocks are for specific version calls. Well, then I'm really confused about how mock.Anything interacts with those ones. I.e., could you just leave this as `conn := &MockConn{}` here since this should just always match